images/linux/scripts/tests/DotnetSDK.Tests.ps1

```diff
@@ -28,4 +28,14 @@ Describe "Dotnet" {
             }
         }
     }
+
+    $dotnetTools = (Get-ToolsetContent).dotnet.tools
+    foreach ($dotnetTool in $dotnetTools) {
+        Context "Dotnet tool $($dotnetTool.name)" {
+            It "dotnet tool $($dotnetTool.name) is available" {
+                $testResult = ($dotnetTool.test)
+                $testResult | Should -Not -BeNullOrEmpty
```

Comment:
We should check exit code, currently, we check that the string is not empty, this statement is always true.

`$dotnetTool.test | Should -ReturnZeroExitCode`

Reply:
OK. Changed.